VTEX - Promotions & Taxes API.json

```diff
@@ -9,7 +9,56 @@
     "servers": [
         {
             "url": "https://{accountName}.{environment}.com.br",
-            "description": "VTEX server url"
+            "variables": {
+                "accountName": {
+                    "description": "Name of the VTEX account. Used as part of the URL",
+                    "default": "apiexamples"
+                },
+                "environment": {
+                    "description": "Environment to use. Used as part of the URL.",
+                    "default": "vtexcommercestable"
+                }
+            }
+        },
+        {
+            "url": "http://{accountName}.{environment}.local/{basePath}",
+            "variables": {
+                "accountName": {
+                    "description": "Name of the VTEX account. Used as part of the URL",
+                    "default": "apiexamples"
+                },
+                "environment": {
+                    "description": "Environment to use. Used as part of the URL.",
+                    "default": "vtexcommercestable"
+                },
+                "basePath": {
+                    "default": "v1"
+                }
+            }
+        },
+        {
+            "url": "https://{accountName}.{environment}.readme.io:{port}/{basePath}",
+            "variables": {
+                "accountName": {
+                    "description": "Name of the VTEX account. Used as part of the URL",
+                    "default": "apiexamples"
+                },
+                "environment": {
+                    "description": "Environment to use. Used as part of the URL.",
+                    "default": "vtexcommercestable"
+                },
+                "port": {
+                    "default": "3000",
+                    "enum": [
+                        "3000",
+                        "5000"
+                    ]
+                },
+                "basePath": {
+                    "description": "path description",
+                    "default": "v1"
+                }
+            }
         }
```

Comment:
Só precisa desse primeiro server! Esses outros 2 são o que está fazendo aparecer essas outras variaveis estranhas no try-it